modules/objdetect/src/qrcode.cpp

```diff
@@ -955,6 +957,8 @@ struct QRCodeDetector::Impl
     ~Impl() {}
 
     double epsX, epsY;
+    vector<vector<Point2f>> qrCorners;
+    vector<vector<Point2f>> alignmentMarkers;
```

Comment:
they should be private or protected.

Reply:
`qrCorners`, `alignmentMarkers` are not public API
`getter` will be unnecessary in my opinion
In public API `qrCorners` return from `QRCodeDetector` by `getQRCorners()`